src/plugins/plugin.legend.js

```diff
@@ -345,14 +346,15 @@ var Legend = Element.extend({
 		var lineWidths = me.lineWidths;
 
 		if (opts.display) {
+			var rtlHelper = getRtlHelper(opts.rtl, me.left, me.minSize.width);
```

Comment:
Why `me.minSize.width`? I think `me.width` should be used instead.

Reply:
According to the comments later in the code that mentions that width is unreliable and could change...

Reply:
Right, that has hopefully been changed since then - but that is another issue and out of the scope of this PR anyway. So `me.minSize` it is.